lib/model/pet.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/cupertino.dart';
+
+class Pet {
+  String uid;
```

Comment:
Please rename this to `id` as `uid` stands for `User ID` and it can be misinterpreted and

Reply:
OK! First, I thought that it refers to 'unique id' :)